### PR TITLE
Feature/article component

### PR DESCRIPTION
**Pull request by Neil-B-Linton**
Hi Team, Please review this PR. 

## Description of Changes:
* Allows user to click a link button on News Stories Component to relocate to the exact Article
* Handles: Linking Articles, Routing, and Embedding the Article website
* Other branches will handle: CSS/SASS, Back button with switch routing, testing
* Rough Scale for future elements above ^^

## Put an 'X' next to all that apply
New Test: []
New Feature:[x]
Bug Fix: [x]

## Before You Submit Ensure
All Tests Pass: [x]
Full Coverage: []
Runs Locally: []

Thanks!

### CODE REVIEW

File: src/components/NewsStories/NewsStories.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ const NewsStories = (event) => {
                 <p className="author-name">by {story.author}</p>
                 <p className="date">{story.date}</p>
               </div>
-              <Link ><button>LINK</button></Link>
+              <Link className="Link__Article" to={{ pathname: "/Article", state: url }}><button>LINK</button></Link>
```
Comment (camianderson) [inline src/components/NewsStories/NewsStories.js]: Great solution!

**Comment by camianderson** [comment]
Good job! Nice to see that Article works now!
Next step is to finish the CSS of the article, back button and testing!

File: src/components/NewsStories/NewsStories.js [hunk 2]
```diff
@@ -51,6 +53,7 @@ const NewsStories = (event) => {
                 <p className="author-name">by {story.author}</p>
                 <p className="date">{story.date}</p>
               </div>
+              <Link className="Link__Article" to={{ pathname: `/Article/${story.title}`, state: story.url }}><button>LINK</button></Link>
```
Comment (jordan-sullivan) [inline src/components/NewsStories/NewsStories.js:56]: perfect solution! -J

**Review by jordan-sullivan — APPROVED** [review]
Great job linking to the articles Neil! Bravo - J

**Pull request merged**